packages/create-contentful-app/template/bin/create-app-definition.js

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+
+const fetch = require('node-fetch');
```

Comment:
Both orgs and app definitions are now supported in contentful-management, we should use it for dogfooding